src/index.js

```diff
@@ -0,0 +1,89 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const cp = require('child_process');
+const util = require('util');
+const fs = require('fs');
+const exec = util.promisify(cp.exec);
+const maven = require('./maven');
+
+async function main() {
+  const release_tag=core.getInput('release-tag');
+  if (!release_tag) return;
+
+  const GITHUB_TOKEN = process.env.github_token;
+  const MULESOFT_NEXUS_USER = process.env.mulesoft_nexus_user;
+  const MULESOFT_NEXUS_PASSWORD = process.env.mulesoft_nexus_password;
+  const secret_key = process.env.secret_key;
```

Comment:
As I mentioned above, let's make these all upper case
```suggestion
  const SECRET_KEY = process.env.SECRET_KEY;
```